kythe/go/extractors/cmd/gotool/analyze_packages.sh

```diff
@@ -17,7 +17,8 @@
 # Download, build, and extract a set of Go packages.  Resulting compilations
 # will be merged into a single .kzip archive in the "$OUTPUT" directory.
 
-: "${TMPDIR:=/tmp}" "${OUTPUT:=/output}"
+: "${TMPDIR:=/tmp}" "${OUTPUT:=/output}" "KYTHE_KZIP_ENCODING=${KYTHE_KZIP_ENCODING:=JSON}"
```

Comment:
Should that be just `"${KYTHE_KZIP_ENCODING:=JSON}"`?

Reply:
yes, thanks